Client/Ecosia/L10N/String.swift

```diff
@@ -182,7 +182,18 @@ extension String {
         case ecosiaLogoAccessibilityLabel = "Ecosia logo"
         case done = "Done"
         case findInPage = "Find in page"
+        case pageControlDots = "Page control dots"
+        case backButtonAccessibilityLabel = "Back button"
         case yourImpactSectionAccessibilityHint = "Open the Your Impact section"
         case yourImpactSectionAccessibilityLabel = "Your Impact section, highlithing the trees planted by yourself and the Ecosia community overall. You have contributed to plant %@ trees. The total number of trees planted by the Ecosia community has reached %@"
+        case onboardingIllustrationTour1 = "This onboarding illustration shows how by performing searches via the Ecosia app, you are leveling up your tree planting impact score. A small search screenshot and a smaller Your Impact section is shown. Leaves on the background."
+        case onboardingIllustrationTour2 = "This onboarding illustration shows briefly an example of a before and after comparision of trees planted in a land. The image is a screenshot from the satellite view."
+        case onboardingIllustrationTour3 = "This onboarding illustration show a few numbers like the projects Ecosia is involved in, the total number of trees planted by the Ecosia community, alongisde the number of countries Ecosia is active. A small map of the planisphere with trees pins in few geographic location, background."
+        case onboardingIllustrationTour4 = "This onboarding illustration is a photo of a monkey climbing a tree. It function mainly as general decoration image."
+        case onboardingSkipTourButtonAccessibilityLabel = "Button that allows you to skip the onboarding"
```

Comment:
Same thing with regards the button's default behaviour here. Might not be the case but, for example, if the content of the button is "skip", it will already automatically voice over "skip button", which for me is a good label already. If not clear, you could then add an accessibility hint describing what the action does.

Like I said, not sure if it applies here, but if so I think the logic could be revised for all buttons, adding a very short accessibility label (potentially even only the button text) and an accessibility hint when not clear. I feel that is closer to what the user would be used in other apps.

Reply:
As some of the buttons have no text (like the back button), I thought about adding complementary info. Will remove it and make it `Skip the onboarding` perhaps.